app.js

```diff
@@ -14,8 +14,8 @@ import {
   getAuthorizationUrlForData,
   getLogoutUrl,
 } from './helpers/utils';
-import { oauthLoginCallback, oauthLogoutCallback } from './controllers/oauthAuthenticationCallback';
-import oauthDataCallback from './controllers/oauthDataCallback';
+import { oauthLoginCallback, oauthLogoutCallback, debugDataFC } from './controllers/oauthAuthenticationCallback';
```

Comment:
As these files now contains other controllers than callback we should process to these renaming:

```
oauthAuthenticationCallback.js => oauthAuthentication.js
oauthDataCallback.js => oauthData.js
```